mitodl/micromasters
# Added flag to enable/disable edx batch updates

#### Pre-Flight checklist


#### What are the relevant tickets?
N/A

#### What's this PR do?
Adds a flag (defaults to `True`) that enables edx batch updates

#### How should this be manually tested?
- Run the tasks normally, the worker should pick them up and perform the operations:

```python
from dashboard import tasks
tasks.batch_update_user_data.delay()
```

- Add `EDX_BATCH_UPDATES_ENABLED=False` to your `.env`
- Run the task again, you should see the task quickly exit and the debug message written to logs (make sure to set `MICROMASTERS_LOG_LEVEL` to at least `DEBUG`
